cli/js/testing.ts

```diff
@@ -185,23 +200,27 @@ export async function runTests({
       `${stats.passed} passed; ${stats.failed} failed; ` +
       `${stats.ignored} ignored; ${stats.measured} measured; ` +
       `${stats.filtered} filtered out ` +
-      `${promptTestTime(suiteEnd - suiteStart)}\n`
+      `${formatDuration(suiteDuration)}\n`
   );
 
-  // TODO(bartlomieju): what's it for? Do we really need, maybe add handler for unhandled
-  // promise to avoid such shenanigans
-  if (stats.failed) {
-    // Use setTimeout to avoid the error being ignored due to unhandled
-    // promise rejections being swallowed.
-    setTimeout((): void => {
-      originalConsole.error(`There were ${stats.failed} test failures.`);
-      testCases
-        .filter(testCase => !!testCase.error)
-        .forEach(testCase => {
-          originalConsole.error(`${RED_BG_FAIL} ${red(testCase.name)}`);
-          originalConsole.error(testCase.error);
-        });
+  // TODO(bartlomieju): is it really needed? Shouldn't that be handled
+  // per test case?
```

Comment:
What does "it" and "that" refer to here?

Reply:
The comment right below; must have inserted empty line by mistake